frontend/src/components/ManageAdminGrid.jsx

```diff
@@ -0,0 +1,26 @@
+import {ManageAdminCard} from "./ManageAdminCard.jsx";
+import {SimpleGrid, useMantineTheme} from "@mantine/core";
+import {useMediaQuery} from "@mantine/hooks";
+
+function ManageAdminGrid({data = [], containerWidth}){
+    const theme = useMantineTheme();
+    const isMobile = useMediaQuery(`(max-width: ${theme.breakpoints.sm})`);
+
+    const cardWidth = isMobile ? 272 : 450;
+    const cardHeight = isMobile ? 70 : 70;
+    const cards = data.map((admin, index) => (
+        <div>
```

Comment:
The key needs to be in the topmost element in the tree, in this case \<div>. Is \<div> even necessary, actually? Anyways, it's still throwing the required key prop error in the console.

Reply:
no need for <div>. fixed.